x-pack/plugins/lens/public/xy_visualization/types.ts

```diff
@@ -43,12 +46,17 @@ export interface XYDataLayerConfig {
   yConfig?: YConfig[];
   splitAccessor?: string;
   palette?: PaletteOutput;
+  yScaleType?: YScaleType;
+  xScaleType?: XScaleType;
+  isHistogram?: boolean;
+  columnToLabel?: string;
 }
 
 export interface XYReferenceLineLayerConfig {
   layerId: string;
   accessors: string[];
-  yConfig?: YConfig[];
+  yConfig?: ExtendedYConfig[];
+  palette?: PaletteOutput;
```

Comment:
Why do we need a palette here?

Reply:
it was always passed here but we don't really need it.

Reply:
Checking right now, but, it seems to me, it is useless, you are right.

Reply:
Removed it.